api/tsconfig.build.json

```diff
@@ -1,4 +1,7 @@
 {
   "extends": "./tsconfig.json",
-  "exclude": ["node_modules", "dist"]
+  "exclude": [
+    "node_modules",
+    "dist"
+  ]
```

Comment:
Changement out-of-scope à enlever (cf point 5 ici https://github.com/horizon-teamdev/HorizonWeb/pull/1#issuecomment-948436489)